app/Filter/BaseDateRangeFilter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Kanboard\Filter;
+
+use Kanboard\Core\DateParser;
+
+/**
+ * Base date filter class
+ *
+ * @package filter
+ * @author  Kamil Ściana
+ */
+abstract class BaseDateRangeFilter extends BaseFilter
+{
+    /**
+     * DateParser object
+     *
+     * @access protected
+     * @var DateParser
+     */
+    protected $dateParser;
+
+    /**
+     * Set DateParser object
+     *
+     * @access public
+     * @param  DateParser $dateParser
+     * @return $this
+     */
+    public function setDateParser(DateParser $dateParser)
+    {
+        $this->dateParser = $dateParser;
+        return $this;
+    }
+
+    /**
+     * Apply a date filter
+     *
+     * @access protected
+     * @param  string $field
+     */
+    protected function applyDateFilter($field)
+    {
+        $dates = explode('..', $this->value);
+        
+        if(count($dates)=== 2){          
+            $timestampFrom = $this->dateParser->getTimestampFromIsoFormat($dates[0]);
+            $timestampTo = $this->dateParser->getTimestampFromIsoFormat($dates[1]);
```

Comment:
To support user defined date format and ISO format, you should use the method `getTimestamp()` from the `DateParser` class instead.

In this way, `createdRange:"2017-12-09..2017-12-15"` and `createdRange:"09/12/2017..15/12/2017"` should work.

Reply:
OK, but in ``BaseDateFilter`` class is used ``getTimestampFromIsoFormat()`` method form the ``DateParser`` class. It was reference for me.